packages/i18n/README.md

```diff
@@ -27,113 +27,37 @@ For a complete example, see the [Internationalization section of the Block Edito
 
 <!-- START TOKEN(Autogenerated API docs) -->
 
-<a name="setLocaleData" href="#setLocaleData">#</a> **setLocaleData**
+<a name="I18n" href="#I18n">#</a> **I18n**
 
-Merges locale data into the Tannin instance by domain. Accepts data in a
-Jed-formatted JSON object shape.
+Instantiable I18n methods
 
-_Related_
+<a name="i18n" href="#i18n">#</a> **i18n**
 
--   <http://messageformat.github.io/Jed/>
+Undocumented declaration.
 
-_Parameters_
+<a name="setLocaleData" href="#setLocaleData">#</a> **setLocaleData**
 
--   _data_ `[LocaleData]`: Locale data configuration.
--   _domain_ `[string]`: Domain for which configuration applies.
+Undocumented declaration.
 
 <a name="sprintf" href="#sprintf">#</a> **sprintf**
 
-Returns a formatted string. If an error occurs in applying the format, the
-original format string is returned.
-
-_Related_
-
--   <http://www.diveintojavascript.com/projects/javascript-sprintf>
-
-_Parameters_
-
--   _format_ `string`: The format of the string to generate.
--   _args_ `...string`: Arguments to apply to the format.
-
-_Returns_
-
--   `string`: The formatted string.
+Internal dependencies
 
 <a name="_n" href="#_n">#</a> **\_n**
 
-Translates and retrieves the singular or plural form based on the supplied
-number.
-
-_Related_
-
--   <https://developer.wordpress.org/reference/functions/_n/>
-
-_Parameters_
-
--   _single_ `string`: The text to be used if the number is singular.
--   _plural_ `string`: The text to be used if the number is plural.
--   _number_ `number`: The number to compare against to use either the singular or plural form.
--   _domain_ `[string]`: Domain to retrieve the translated text.
-
-_Returns_
-
--   `string`: The translated singular or plural form.
+Undocumented declaration.
```

Comment:
This loss of information is highly problematic to me. I believe that a combination of @jsnajdr 's comment at https://github.com/WordPress/gutenberg/pull/20318/files#r383723605 and an addition of JSDoc to methods in `default-i18n.js` should restore the documentation.